.github/workflows/node.js.yml

```diff
@@ -68,7 +69,9 @@ jobs:
         run: yarn install
 
       - name: 🏗️ Build app
-        run: yarn build --configuration production
+        run: yarn build --configuration $buildEnvironment
+        env: 
+          buildEnvironment: ${{ github.ref == 'refs/heads/development' && 'staging' || 'production' }}
```

Comment:
should `staging` here be `development`?

Reply:
Good catch, I altered it on the development branch and not on this PR. Should be fixed now.